src/core/client/admin/components/UserHistoryDrawer/UserDrawerAccountHistory.tsx

```diff
@@ -117,6 +130,19 @@ const UserDrawerAccountHistory: FunctionComponent<Props> = ({ user }) => {
       });
     });
 
+    user.status.username.history.forEach((record, i) => {
+      history.push({
+        kind: "username",
+        action: {
+          username: record.username,
+          prevUsername:
+            i >= 1 ? user.status.username.history[i - 1].username : null,
```

Comment:
I'd add a comment here as to the logic being performed. After reading it, I know it's grabbing the previous username, but it would be helpful to have it documented here.

Reply:
Alternatively, if you don't like a comment, a neatly named function clarifying what it's doing could be good too.